pom.xml

```diff
@@ -1173,86 +1173,10 @@
-            <!-- Excluded due to Maven Cobertura Plugin's issue #5: https://github.com/mojohaus/cobertura-maven-plugin/issues/5-->
-            <excludedLink>http://mojo.codehaus.org/cobertura-maven-plugin/</excludedLink>
-            <!-- Excluded due to Maven JDepend Plugin's issue #2: https://github.com/mojohaus/jdepend-maven-plugin/issues/2-->
-            <excludedLink>http://mojo.codehaus.org/jdepend-maven-plugin</excludedLink>
-            <!-- Excluded due to Maven Taglist Plugin's issue #3: https://github.com/mojohaus/taglist-maven-plugin/issues/3-->
-            <excludedLink>http://mojo.codehaus.org/taglist-maven-plugin</excludedLink>
-            <!-- Excluded due to Maven XML Plugin's issue #1: https://github.com/mojohaus/xml-maven-plugin/issues/1-->
-            <excludedLink>http://mojo.codehaus.org/xml-maven-plugin</excludedLink>
-            <!-- Excluded due to Maven ANTLR4 Plugin's issue #978: https://github.com/antlr/antlr4/issues/978-->
-            <excludedLink>http://www.antlr.org/antlr4-maven-plugin</excludedLink>
-            <!-- till https://github.com/mojohaus/versions-maven-plugin/pull/49 -->
-            <excludedLink>http://mojo.codehaus.org/versions-maven-plugin</excludedLink>
-            <!-- Excluded due to Maven Release Plugin's issue #919: https://issues.apache.org/jira/browse/MRELEASE-919-->
-            <excludedLink>http://maven.apache.org/plugins/maven-release-plugin/</excludedLink>
-            <!-- Excluded till https://issues.sonatype.org/browse/NEXUS-9643 -->
-            <excludedLink>http://www.sonatype.com/public-parent/nexus-maven-plugins/nexus-staging/nexus-staging-maven-plugin/</excludedLink>
-            <!-- Excluded till https://github.com/mojohaus/build-helper-maven-plugin/issues/3 -->
-            <excludedLink>http://www.mojohaus.org/build-helper-maven-plugin/build-helper-maven-plugin</excludedLink>
-            <!-- This link exists, but it has too much content and github has problems with rendering it quickly, so to avoid ocasional "Read timed out" we exclude it -->
-            <excludedLink>https://github.com/checkstyle/checkstyle/issues/1555</excludedLink>
-            <!-- Excluded, because the link is generated by sonar's maven report plugin and we
-            can not correct the link while it will not be changed in effective pom.xml of the plugin
-            by sonar's developers. It is not allowed to open issue against sonar's maven report plugin
-            at sonar's issue tracker: http://jira.codehaus.org/browse/MSONAR
-            Sonar's mailing lists: http://www.mojohaus.org/sonar-maven-plugin/mail-lists.html-->
-            <excludedLink>https://sonarqube.com/project/index/com.puppycrawl.tools:checkstyle</excludedLink>
-            <!-- skip validating SNAPSHOTs on sourceforge when ${projectVersion} is SNAPSHOT-->
-            <excludedLink>http://downloads.sourceforge.net/project/checkstyle/checkstyle/*-SNAPSHOT/checkstyle-*-SNAPSHOT-all.jar</excludedLink>
```

Comment:
something is different ... no it is possible to remove that excludes as even original plugin does not report this as problem even it is clear redirection of 404 to another page .
http://wheregoes.com/retracer.php for "http://downloads.sourceforge.net/project/checkstyle/checkstyle/7.7-SNAPSHOT/checkstyle-7.7-SNAPSHOT-all.jar" show a problem but in browser this link is redirected to https://sourceforge.net/projects/checkstyle/files/ , even redirections a lot allowed in config.